Changelog.md

```diff
@@ -29,6 +29,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added `VK_EXT_hdr_metadata` device extension (#804)
 - Added `VK_NV_cuda_kernel_launch` device extension (#805)
 - Added `descriptor_count()` setter on `ash::vk::WriteDescriptorSet` (#809)
+- Added `*_as_c_str()` getters for `c_char` pointers and `c_char` arrays (#831)
```

Comment:
Nit in hindsight: we also added _setters_ for `c_char` pointers... And only replaced the `c_char` **array** setters with a `CStr` interface.